main/CommandHandler.cpp

```diff
@@ -85,6 +85,13 @@ NetworkUDP udps[MAX_SOCKETS];
 NetworkServer tcpServers[MAX_SOCKETS];
 NetworkClientSecure tlsClients[MAX_SOCKETS];
 
+// Root certificate bundle provided by ESP-IDF. This available by default via ESP-iDF esp_crt_bundle_attach(),
+// but arduino-esp32 doesn't provide a way to invoke that default easily, so we declare these to use
+// the default bundle explicitly.
+extern const uint8_t x509_crt_imported_bundle_bin_start[] asm("_binary_x509_crt_bundle_start");
```

Comment:
@dhalbert seems like we don't need to use combine.py to convert rootcert to binary + written it to NVS keys anymore ? 

```
I (60) boot: Partition Table:
I (64) boot: ## Label            Usage          Type ST Offset   Length
I (71) boot:  0 nvs              WiFi data        01 02 00009000 00006000
I (79) boot:  1 phy_init         RF data          01 01 0000f000 00001000
I (86) boot:  2 certs            NVS keys         01 04 00010000 00020000
I (94) boot:  3 factory          factory app      00 00 00030000 00180000
I (101) boot:  4 storage          Unknown data     01 82 001b0000 00040000
I (109) boot: End of partition table
```